typegraph/core/wit/typegraph.wit

```diff
@@ -15,12 +15,14 @@ interface core {
     finalize-typegraph: func() -> result<string, error>
 
     type type-id = u32
-
     record type-base {
         name: option<string>,
+        // json
+        injection: option<string>,
         // string => json string
         runtime-config: option<list<tuple<string, string>>>
     }
+    update-type-injection: func(id: u32, value: string) -> result<_, error>
```

Comment:
I would rather not modify the type, as the same type might be used elsewhere without the injection. See the approach I used for policies.

```
record type-with-injection {
   tpe: type-id,
   injection: ...
}
```